api.go

```diff
@@ -1,5 +1,5 @@
 // Package livepeer API
-package livepeerAPI
+package lp_api
```

Comment:
should this be just named `api`?

Reply:
the livepeer part would be apparent from `github.com/livepeer` part, and usage/imports can rename it anyway!

Reply:
Right... I was reading on [go guidelines](https://go.dev/blog/package-names) and it's actually an anti-pattern to add `_` in the package names... 

Although they also suggest to avoid too generic package names like "api" which could easily have name clashes with any other API client package. 

So I was kinda split here, I don't really like `lp_api` but I think it's an improvement from `livepeerAPI` (guidelines also unwelcome mixed casing). `lpapi` looks weird as I can only read "papi" 😆 and `lapi`, an abbreviation used in `task-runner`, also feels too unspecific. 

So idk. I'm open to switching to just `api` as well, might be better and as you said importers can rename on clashes. Is this still your preferred choice from the above?

Reply:
OK made my mind. Switched to just `api` 👍